automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/DownloadUtil.scala

```diff
@@ -68,14 +68,16 @@ trait DownloadUtil extends Eventually with LazyLogging with WebBrowserUtil {
   private def downloadFile(downloadPath: String, fileName: String, downloadThing: Either[Clickable, CssSelectorQuery])(implicit webDriver: WebDriver): String = synchronized {
 
     def archiveDownloadedFile(sourcePath: String): String = {
-      // wait up to 10 seconds for file exist
+      // wait up to 30 seconds for file exist
```

Comment:
occasionally, test failed to detect download file in 10 seconds. combining small change in this PR.